module-services/service-evtmgr/EventManager.cpp

```diff
@@ -321,6 +321,10 @@ int EventManagerCommon::dumpLogsToFile()
 
 void EventManagerCommon::handleMinuteUpdate(time_t timestamp)
 {
+    // temporary frequency lock to handle time update event
+    if (cpuSentinel) {
+        cpuSentinel->HoldMinimumFrequencyForTime(bsp::CpuFrequencyMHz::Level_4, std::chrono::seconds(1));
```

Comment:
Isn't second too long? I believe it usually takes around ~300ms of processing to handle update. Won't keeping the sentinel for longer have negative effect on battery usage?

Reply:
Immediately after processing the data (~ 300ms), a drawing on the screen takes place, which lasts ~ 5 seconds, so this 1 second reserve has no effect because at this time the eInk sentinel will be active.